include/swift/AST/Decl.h

```diff
@@ -6266,7 +6266,8 @@ class AbstractFunctionDecl : public GenericContext, public ValueDecl {
 
 public:
   /// Get all derivative function configurations.
-  ArrayRef<AutoDiffConfig> getDerivativeFunctionConfigurations();
+  ArrayRef<AutoDiffConfig>
```

Comment:
Could you please document the `NonPrimaryLookup` parameter? That would be helpful as its meaning is not obvious.

How about renaming to something like `lookInNonPrimarySources`? I think that would be clearer and more fluent.